### DISCUSSION TITLE
saveToS3 return value?

### DISCUSSION THREAD

**Discussion by bryanjamesmiller** [stefanzweifel/sidecar-browsershot#23] [Ideas]
Hey @stefanzweifel - probably my last question for you :) thanks for all the discussion - your awesome new `saveToS3` method works and I see my screenshot in my S3 bucket, but I seem to be getting only `null` returned in `$returnValue` after calling:

`$returnValue = BrowsershotLambda::url('some-url')->saveToS3('some-custom-file.png', 'bucket-name');`

Is there a way the `$returnValue` can include data that confirms that the screenshot was saved successfully to S3?

**Comment by stefanzweifel** [comment]
Hi Bryan

Yeah that's doable. I forgot to return the return value we get from S3. I've prepared a change in #24 and will tag a new version shortly.
